src/WordTokenizers.jl

```diff
@@ -1,20 +1,21 @@
+
 module WordTokenizers
 
 export poormans_tokenize, punctuation_space_tokenize,
        penn_tokenize, improved_penn_tokenize, nltk_word_tokenize,
        tokenize,
        rulebased_split_sentences,
        split_sentences,
-       set_tokenizer, set_sentence_splitter
+       set_tokenizer, set_sentence_splitter,
+       rev_tokenize, rev_detokenize
 
-include("words/fast.jl")
```

Comment:
I am guessing this was deleted by mistake during a rebase.
Removing it is why tests are now failing